academy/ibc/migration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "IBC Migration"
+order: 
+description: 
+tag: deep-dive
+---
+
+# IBC Migration
```

Comment:
link to this repo somewhere: https://github.com/cosmos/ibc-go/tree/main/docs/migrations

Reply:
also, if we are talking about migrations, do we need to also talk about integrating ibc-go into the rest of the SDK? or is that covered already in the SDK part of the Academy? https://github.com/cosmos/ibc-go/blob/main/docs/ibc/integration.md